backend/controllers/autoCompleteController.js

```diff
@@ -73,7 +73,7 @@ addInterest = async function (req, res) {
             addedInterest = await Interest.create({interest: interest})
             res.status(200).send({message: "Interest is created", interest: addedInterest.interest})
         } catch (error) {
-            res.status(500).send(error)
+            res.status(500).send(error.message)
         }
     }
     return res.status(404).send({message: "Erroronous Input"})
```

Comment:
Fix

Reply:
Btw these returns work fine.
When the input is wrong, the code does not go into if conditions and ends with return.
You can try it.

Reply:
When the input is not wrong, then it throws an ERROR. They are not fine. 

Is this your definition of fine? Why are you insisting on it, I have told you this yesterday.
![Error](https://user-images.githubusercontent.com/44070728/105206364-574eda80-5b57-11eb-9058-2ca6ae5ca99e.png)

Reply:
And 404 HTTP STATUS CODE is also wrong, it should be 400

Reply:
Yeah in the postman it works fine but it seems it gives error on the code, removing returns.
![image](https://user-images.githubusercontent.com/44067394/105209384-d09bfc80-5b5a-11eb-834f-cd5d370c9cc1.png)

Reply:
Thanks a lot, other than those it seems great. I believe it is safe to merge when you fix them.

Reply:
Thanks! Other than those great job.

Reply:
@EnesToptas btw do you still see these commits?
It seems they were created because the initial merge request was into master
Currently clicking on the file name takes you nowhere, it seems they are outdated.

Reply:
Yes probably, but you can still fix them on your local then push to this branch. Autocomplete endpoints will be used heavily and I believe those errors are important.

Reply:
I have problems accessing to the files and the branch, currently my repo can not see the this branch,

Creating a new branch to remove returns from autocomplete endpoints, we can merge this branch.

Reply:
use git fetch please.

Reply:
Nope it does not show with git fetch either:

![image](https://user-images.githubusercontent.com/44067394/105211647-92eca300-5b5d-11eb-8919-691be21df3f6.png)

I can not access to current version of the file in this branch, could you fix the file?